var/spack/repos/builtin/packages/sensei/package.py

```diff
@@ -87,7 +93,8 @@ def cmake_args(self):
         if '+libsim' in spec:
             args.extend([
                 '-DENABLE_LIBSIM:BOOL=ON',
-                '-DVISIT_DIR:PATH={0}'.format(spec['visit'].prefix),
+                '-DVISIT_DIR:PATH={}/current/{}-{}'.format(
```

Comment:
FWIW. This is likely the line causing the validation check problem.  I recommend adding numbers within the curly braces (starting with '0').

Reply:
Thanks for the suggestion. I'll do that.